plugins/lua/stockpiles.lua

```diff
@@ -447,28 +447,49 @@ function StockpilesOverlay:init()
         frame_background=gui.CLEAR_PEN,
         visible=is_expanded,
         subviews={
-            -- widgets.HotkeyLabel{
-            --     frame={t=0, l=0},
-            --     label='import settings',
+            -- widgets.Label{
+            --     frame={t=0, l=0, h=1},
+            --     auto_height=false,
+            --     text={'Stockpile Settings:'},
+            --     text_pen=COLOR_DARKGREY,
+            -- },  widgets.HotkeyLabel{
+            --     frame={t=1, l=0},
+            --     label='Copy',
             --     auto_width=true,
-            --     key='CUSTOM_CTRL_I',
+            --     key='CUSTOM_CTRL_C',
+            --     option_gap=-1, -- option_gap doesn't seem to be a feature of widgets.HotkeyLabel
+            --     on_activate=do_export,
+            -- },  widgets.HotkeyLabel{
+            --     frame={t=1, l=16},
+            --     label='Paste',
+            --     auto_width=true,
+            --     key='CUSTOM_CTRL_V',
+            --     option_gap=-1,
             --     on_activate=do_import,
-            -- }, widgets.HotkeyLabel{
-            --     frame={t=1, l=0},
-            --     label='export settings',
+            -- },  widgets.HotkeyLabel{
+            --     frame={t=1, l=32},
+            --     label='Export',
             --     auto_width=true,
             --     key='CUSTOM_CTRL_E',
+            --     option_gap=-1,
             --     on_activate=do_export,
+            -- },  widgets.HotkeyLabel{
+            --     frame={t=1, l=48},
+            --     label='Import',
+            --     auto_width=true,
+            --     key='CUSTOM_CTRL_I',
+            --     option_gap=-1,
+            --     on_activate=do_import,
             -- },
             widgets.Panel{
                 frame={t=0, l=0},
                 subviews={
                     widgets.Label{
                         frame={t=0, l=0, h=1},
                         auto_height=false,
-                        text={'Designate items/animals brought to this stockpile for:'},
+                        text={'Automatically designate items/animals in this stockpile'},
```

Comment:
The grammar of existing options doesn't line up. "Automatically designate items/animals in this stockpile Training" doesn't parse. What is wrong with the original wording?

Reply:
You're right, I didn't feel strongly about this and will switch it back. I was playing with a few things when doing the layout because I was trying to come up with a way for it to work with some version of forbid/forbidden since I similarly wasn't liking it grammatically for the new option ("Designate items for unforbid/forbidden"). I had thought I was getting somewhere with "Automatically" because I kept thinking of auto-dump, auto-forbid, etc. But I've come up short and will keep the original language.